### ISSUE TITLE
Experience drops should be forced on if "Experience Drops" plugin is enabled

### ISSUE THREAD

**Issue by Artamus** [runelite/runelite#632]
- **Overview of the issue** - currently the *Experience Drops* plugin only offers colouring options for different prayers. If the vanilla client *XP drops* is turned off entirely, the plugin should still be able to override the setting for the actual experience drops themselves and enable them. 

In addition, I personally feel like *XP Globes* and *Experience Drops* plugins could and should be combined to something slightly more user friendly

- **Java version and operating system** - Windows 10 with a JRE from the game client (1.2.16 w/ OldSchool Engine Rev 163)

- **Reproduce the error** - not really an *error*, so just run the client and it's there in the plugin list

- **Related issues** - None

- **Suggest a fix** - Perhaps merge the plugins into one plugin, name it *XP Globes/Drops*, it would have the following menu options: Enable, Enable Tooltips, Force Enable Drops, Enable Coloured Drops, the 3 current colour options under XP drops for the prayers

**Comment by devLotto** [comment]
From what I've found, the XP drops feature is enabled by the client sending a packet to the server when you click on the orb. We can do the same and get the desired result, but I don't know if it's safe to do since it entails interacting with interfaces without the user's action.

**Comment by Adam-** [comment]
The packet is just for the interface click. It is likely enabled via some varbit.

**Comment by devLotto** [comment]
Nope. The server sends that the client should show/hide the XP drop widget on login and on clicking the orb. Preventing any setting of values in the "settings" array does not keep the XP drops/experience bar from being toggled.

**Comment by devLotto** [comment]
We could maybe roleplay the server and display the XP drops widget ourselves though..
